pyEpiabm/pyEpiabm/core/cell.py

```diff
@@ -6,12 +6,13 @@
 import numpy as np
 from queue import Queue
 from numbers import Number
+import re
```

Comment:
```suggestion
import numpy as np
import re
from queue import Queue
from numbers import Number
```

Prefer module imports before class imports